### PR TITLE
Fixes creation of amendments

### LINKED ISSUE #195: Motions: Create amendment currently does not work

### LINKED ISSUE #215: Reload while creating an amendment will not load its parent
Reloading the editing view while a user creates an amendment will not reload its parent.
Try: Create amendment -> press `F5`

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
Fixes #195 
Fixes #215 
Fixes [openslides-backend#696](https://github.com/OpenSlides/openslides-backend/issues/696)

Waiting for #190

### CODE REVIEW

**Comment by tsiegleauq** [comment]
> Waiting for #187

Why? This seems unrelated.

**Review by FinnStutzenstein — APPROVED** [review]
Looks good, not tested

**Comment by GabrielInTheWorld** [comment]
> > Waiting for #187
> 
> Why? This seems unrelated.

I meant #190: [active-meeting-id.service.ts#L35](https://github.com/OpenSlides/openslides-client/blob/0a018e1c60252540d4e62abc5c05a5449c76f54c/client/src/app/core/core-services/active-meeting-id.service.ts#L35) has an error, which prevents the creation of an amendment. #190 has a fix for that.

**Review by tsiegleauq — CHANGES_REQUESTED** [review]
Paragraph based amendments are almost fine.
Text changes are not saved or rendered in the motion.
In Settings, set "how to create new amendments" to "Paragraph-based, Diff-enabled".

Create a new Amendment of motion/1
Select a paragraph and change

The parent was set just fine,
It appears as an amendment to motion/1,
The text changes show : "No changes at the text." (in italic)

using **"Empty text field"**:
"new-amendment?parent=1" was set just fine.
Saving the new amendment was submitting a new motion that was not regisered to be an amendment. The detail view does not show the parent, the supposed-to-be parent does not show the amendment

using: **"Edit the whole motion text"**
Same as above, but the motion text of the parent was missing in tinyMCE

**Review by tsiegleauq — CHANGES_REQUESTED** [review]
There is still something really odd with paragraph based amendments.

* updating (changing) a paragraph based amendment does not work. The save button is always disabled
* in the view of a paragraph based amendment: switching between line number modes **duplicates the text**. That one is really odd.

before:
image

changing line number outside once:
image

changing back to no line numbers:
image

changing to outside again:
image

**Comment by GabrielInTheWorld** [comment]
@tsiegleauq Please re-test and let me know, if everything is okay or anything is missing.

File: client/src/app/site/motions/modules/motion-detail/components/base/base-motion-detail-child.component.ts [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+import { ViewUnifiedChange } from 'app/shared/models/motions/view-unified-change';
+import { BaseComponent } from 'app/site/base/components/base.component';
+import { ViewMotion } from 'app/site/motions/models/view-motion';
+import { ViewMotionChangeRecommendation } from 'app/site/motions/models/view-motion-change-recommendation';
+import { ChangeRecoMode, LineNumberingMode } from 'app/site/motions/motions.constants';
+import { MotionServiceCollectorService } from '../../../services/motion-service-collector.service';
+import { MotionViewService } from '../../../services/motion-view.service';
+
+@Directive()
+export class BaseMotionDetailChildComponent extends BaseComponent {
```
Comment (tsiegleauq) [inline client/src/app/site/motions/modules/motion-detail/components/base/base-motion-detail-child.component.ts:26]: base class directive should probably be abstract if you never plan to actually make it an instance or a component

**Review by tsiegleauq — APPROVED** [review]
Works as expected. There were other unrelated (non breaking) bugs that we might take care of later. Not entirely sure if the structure is optimal yet, especially regarding `base-motion-detail-child.component.ts `, but we might tackle this another time. Can be rebased and merged.

**Pull request merged**